sys/abi/src/lib.rs

```diff
@@ -252,7 +252,10 @@ impl TaskState {
     /// Checks if a task in this state is trying to deliver a message to
     /// `target`.
     pub fn is_sending_to(&self, target: TaskId) -> bool {
-        self == &TaskState::Healthy(SchedState::InSend(target))
+        match self {
+            TaskState::Healthy(SchedState::InSend(t)) => *t == target,
```

Comment:
Using matches instead of generating whole `PartialEq`/`Eq` implementations is a smart move when optimizing for binary size, i'll have to remember that.